hack/make-rules/test-cmd-util.sh

```diff
@@ -1253,7 +1253,7 @@ run_kubectl_run_tests() {
   # Post-Condition: Job "pi" is created
   kube::test::get_object_assert jobs "{{range.items}}{{$id_field}}:{{end}}" 'pi:'
   # Describe command (resource only) should print detailed information
-  kube::test::describe_resource_assert pods "Name:" "Image:" "Node:" "Labels:" "Status:" "Created By"
+  kube::test::describe_resource_assert pods "Name:" "Image:" "Node:" "Labels:" "Status:"
```

Comment:
For every `kube::test::describe_resource_assert` call to a resource that's controlled by something, verify that "Controlled By" is there too

Reply:
Does this include:
https://github.com/kubernetes/kubernetes/blob/master/hack/make-rules/test-cmd-util.sh#L520
https://github.com/kubernetes/kubernetes/blob/master/hack/make-rules/test-cmd-util.sh#L3983

Reply:
As synced by person, the above two tests should not be added with "Controlled By" print attribute. For the first test, we are not interested with it. For the second one, although a node is managed by node controller, "Controlled By" print attribute will look at `controllerRef` field, which is not available for nodes.

I also did a thorough search, and added "Controlled By" to tests that manage to pass with it.